strings-csharp/ReplaceLineBreaksInAStringCSharp/ReplaceLineBreaksInAStringCSharp/ReplaceLineBrake.cs

```diff
@@ -0,0 +1,27 @@
+using System.Text.RegularExpressions;
+
+namespace ReplaceLineBreaksInAStringCSharp;
+
+public static class ReplaceLineBrake
```

Comment:
@koddek 
I think the name you meant here is `ReplaceLineBreak`. This change applies to both the method name and the file name